From: Larry Finger <Larry.Finger@lwfinger.net>
Subject: [PATCH V2] staging: r8712u: Add new USB ID
Date: Sun, 11 Dec 2011 10:27:54 -0600
Message-ID: <4ee4da0a.gJM5XfxwTWfv6h1X%Larry.Finger@lwfinger.net>
Cc: devel@driverdev.osuosl.org, post@rolandgruber.de
To: Greg Kroah-Hartman <gregkh@suse.de>

Add USB ID for Sitecom WLA-2000 v1.001 WLAN.

Reported-and-tested-by: Roland Gruber <post@rolandgruber.de>
Signed-off-by: Larry Finger <Larry.Finger@lwfinger.net>
Cc: Stable <stable@vger.kernel.org>
---

V2 - fix typo in ID.

Index: staging/drivers/staging/rtl8712/usb_intf.c
===================================================================
--- staging.orig/drivers/staging/rtl8712/usb_intf.c
+++ staging/drivers/staging/rtl8712/usb_intf.c
@@ -89,6 +89,7 @@ static struct usb_device_id rtl871x_usb_
 	{USB_DEVICE(0x0DF6, 0x0045)},
 	{USB_DEVICE(0x0DF6, 0x0059)}, /* 11n mode disable */
 	{USB_DEVICE(0x0DF6, 0x004B)},
+	{USB_DEVICE(0x0DF6, 0x005D)},
 	{USB_DEVICE(0x0DF6, 0x0063)},
 	/* Sweex */
 	{USB_DEVICE(0x177F, 0x0154)},
